### PR TITLE
fix(droppable): Bind drag drop handlers when wrapped element changes

### PR DESCRIPTION
**Pull request by baipenghan**
This change fixes the issue that when Wrapped Component changes itself, all the drag and drop event listeners are added to the new element, thus solving the Ghost State issue where event listeners are attached to Ghost State Component first but not attached to FilesPage Component

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @baipenghan has signed the CLA. Thanks for the pull request!

**Comment by cweeii** [comment]
Can you make sure there are spaces between the arrange/act/assert sections?  It's a little difficult to read the tests when they're together.

File: src/elements/common/droppable/makeDroppable.js [hunk 1]
```diff
@@ -62,22 +62,37 @@ const makeDroppable = ({ dropValidator, onDrop }: { dropValidator?: Function, on
...
+         * event listeners on the new droppableEl
          */
         bindDragDropHandlers = () => {
             const droppableEl = findDOMNode(this); // eslint-disable-line react/no-find-dom-node
-            if (!droppableEl || !(droppableEl instanceof Element)) {
+            if (!droppableEl || !(droppableEl instanceof Element) || droppableEl === this.droppableEl) {
                 return;
             }
 
+            if (this.droppableEl) {
```
Comment (jstoffan) [inline src/elements/common/droppable/makeDroppable.js]: Can we move this to `componentDidUpdate`, since it's only relevant for that lifecycle method?

Comment (baipenghan) [reply src/elements/common/droppable/makeDroppable.js]: Sure. makes sense

**Pull request merged**